luma/core/device.py

```diff
@@ -168,6 +173,62 @@ def display(self, image):
         :param image: Image to display.
         :type image: PIL.Image.Image
         """
-        assert(image.size == self.size)
+        assert image.size == self.size
 
         self.image = self.preprocess(image).copy()
+
+
+class fb(device):
+    def __init__(self, display=None, rotate=0, **kwargs):
+        super(fb, self).__init__(serial_interface=noop())
+        self.id = self.__get_display_id(display)
+        (width, height) = self.__config("virtual_size")
+        self.bits_per_pixel = next(self.__config("bits_per_pixel"))
+        assert self.bits_per_pixel in [24, 32]
```

Comment:
what are these magic nrs here? Can you assign a meaningful var name to this 24, 32?

Reply:
24 relates to a bit depth of 24 - basically enough to store RGB colours in 3 bytes per pixel
32 is RGBA - the extra byte is the alpha channel, which needs 4 bytes per pixel

Reply:
to me, sounds like a good comment to add

Reply:
refactored code